### ISSUE TITLE
Trap focus using arrow keys on accordion triggers

### ISSUE THREAD

**Issue by geotrev** [geotrev/undernet#143]
A group of accordion triggers will not retain focus using arrow keys, currently.

According to WCAG standard, this should be present on all accordions. This is doable using `ContextUtil` by creating a new focus trap instance on each accordion component instance on a page. 💅
